bash_completion

```diff
@@ -113,32 +115,38 @@ _sysvdirs()
     return 0
 }
 
+_comp_deprecate_func _sysvdirs _comp_sysvdirs
+
 # This function checks whether we have a given program on the system.
 #
-_have()
+_comp_has_command()
 {
     # Completions for system administrator commands are installed as well in
     # case completion is attempted via `sudo command ...'.
     PATH=$PATH:/usr/sbin:/sbin:/usr/local/sbin type $1 &>/dev/null
 }
 
+_comp_deprecate_func _have _comp_has_command
+
 # Backwards compatibility for compat completions that use have().
 # @deprecated should no longer be used; generally not needed with dynamically
-#             loaded completions, and _have is suitable for runtime use.
+#             loaded completions, and _comp_has_command is suitable for runtime use.
 have()
 {
     unset -v have
-    _have $1 && have=yes
+    _comp_has_command $1 && have=yes
 }
 
 # This function checks whether a given readline variable
 # is `on'.
 #
-_rl_enabled()
+_comp_rl_enabled()
```

Comment:
I was contemplating `_comp_is_rl_enabled` for this, but I think it's better as `_comp_rl_enabled`.

Then again there's little reason to keep it this short, perhaps `_comp_readline_variable_on` or `_comp_readline_variable_enabled` (or "variable" shortened to "var" in either) would be better.

Again, leaving up to you to decide.

Reply:
I adopted `_comp_readline_variable_on`. <!-- because `enabled` does not always apply as some variables are not necessarily `on` or `off` but can contain arbitrary strings -->